french-onion-soup-starter-code/css/main.css

```diff
@@ -0,0 +1,90 @@
+/*Some necessary adjustments*/
+h1, body, html, main {
+  margin: 0;
+}
+/*Import the font and set the default size*/
+html {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px
+}
```

Comment:
Don't mind to use our [boilerplate](https://github.com/ironhack/bcn-webdev-cheatsheet/blob/master/m1/css/boilerplate.css)